registry/remote/repository_test.go

```diff
@@ -2210,6 +2210,98 @@ func Test_BlobStore_Resolve(t *testing.T) {
 	}
 }
 
+func Test_Manifest_generateManifestDescriptorWithVariousDockerContentDigestHeaders(t *testing.T) {
```

Comment:
If this is to test the `generateManifestDescriptor` method, we should call it ` Test_generateManifestDescriptor`.

Reply:
Do we want to also test the `generateBlobDescriptor` method?

Reply:
Yeah the name was bad; since I've made it a method `(s *manifestStore)` again (to access `s.repo.MaxMetadataBytes`, the test is now called `Test_ManifestStore_generateDescriptorWithVariousDockerContentDigestHeaders`

Reply:
Let me take a look at `generateBlobDescriptor` and add some tests.

Reply:
This is done and **`R4R`**